include/vcpkg/commands.remove.h

```diff
@@ -13,10 +17,10 @@ namespace vcpkg::Remove
         YES
     };
 
```

Comment:
I know it's not part of this PR, but should this command also be moved to the Commands namespace/ follow the new convention for the "perform_and_exit"?

Reply:
I think all such changes should be done in their own PR